routes/v1.js

```diff
@@ -63,7 +64,12 @@ function addRoutes(app, peliasConfig) {
     search: createRouter([
       sanitisers.search.middleware,
       middleware.calcSize(),
-      controllers.search(peliasConfig),
+      // 2nd parameter is `backend` which gets initialized internally
+      // 3rd parameter is which query module to use, use fallback/geodisambiguation
+      //  first, then use original search strategy if first query didn't return anything
+      controllers.search(peliasConfig, undefined, require('../query/search')),
```

Comment:
if you could move the `require` statements from line 70 and 72 into an object called `search_strategy` or something to that effect, it would be more in line with the rest of this file and make it easier to read the code.